docs/resources/notebook.md

```diff
@@ -0,0 +1,73 @@
+# databricks_notebook Resource
+
+This resource allows you to manage the import, export, and delete notebooks. The maximum allowed size of a 
+request to resource is 10MB. 
+
+-> **Note** Though the public workspace import api supports notebooks of type `DBC`, `JUPYTER` and `HTML` this resource does not support 
+those types as determining changes is quite challenging as the format includes additional information such as timestamps 
+execution counts thus a base64 diff causes issues.
+
+
+## Example Usage
+
+For deployment of an empty Python notebook, the following example might be useful:
+
+```hcl
+resource "databricks_notebook" "notebook" {
+  content = base64encode("# Welcome to your Python notebook")
+  path = "/mynotebook"
+  overwrite = false
+  mkdirs = true
+  language = "PYTHON"
+  format = "SOURCE"
+}
+```
+    
+## Argument Reference
+
+The following arguments are supported:
+
+#### - `content`:
```

Comment:
perhaps in a separate PR, but this should be formatted according to registry docs